docs/getting-started/github-apps/github-apps.rst

```diff
@@ -270,28 +293,7 @@ filters
         Pattern-strings use `Unix-style Glob syntax <https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/FileSystem.html#getPathMatcher(java.lang.String)>`_ by default (Ex: ``develop``, ``myworkflow/**``),
         or RegEx when the string is surrounded by ``/`` (Ex: ``/develop/``, ``/myworkflow\/.*/``).
 
-It's important to note that we originally released our services tutorial using version 1.1 of the ``/.dockstore.yml`` file. For more info on
-services and registering them, check out our :doc:`Getting Started with Services </getting-started/getting-started-with-services>` which has been updated to use 1.2.
-
-
-Error Handling
-----------------------------------
-Since Dockstore relies on GitHub to tell us when changes have been made on GitHub, there are chances that the message gets lost or delayed.
-Typically, Dockstore reacts within seconds of a change being made on GitHub, however service disruptions can delay this to a few minutes.
-If a message were to get lost, unfortunately you will need to push to GitHub again. Currently, there is no way to tell on Dockstore whether
-a GitHub message was delayed or lost. We recommend waiting a few minutes and then trying to push again. This will be changed in the future.
-
-Another error that could occur is that we received the message from GitHub, however the ``/.dockstore.yml`` is invalid. If we cannot read the 
-file, then we do not know which workflow or service to associate the error with. For now, please ensure that your file is a valid YAML file and
-compare it with our examples/documentation to confirm that you filled it in correctly. In the future we plan to have a system in place where
-users can keep track of these GitHub events and resulting action taken by Dockstore, even if the message was succesfully handled.
-
-Another possible issue is that we received the message from GitHub, but the user who triggered the message event is not registered on Dockstore with
-the corresponding GitHub account. This is only an issue if the workflow or service does not already exist on Dockstore. When creating new workflows and
-services, we need to be able to associate them with a user. If the workflow or service already exists on Dockstore, then this error will not occur and the 
-version will be properly added/updated/deleted on Dockstore.
-
-As always, you can reach out to our team on our `discussion forum <https://discuss.dockstore.org/>`_ to discuss any issues you are facing.
+For more info on services and registering them, check out our :doc:`Getting Started with Services </getting-started/getting-started-with-services>`, or our :doc:`Service Version 1.2 Template</assets/templates/template.html#service-version-1-2-template>`.
```

Comment:
Do you need the part starting with `or our :doc:...`? If so, it's not a valid link and gives a build warning.

Reply:
I'm trying to link to https://docs.dockstore.org/en/develop/assets/templates/template.html#service-version-1-2-template, what is the correct way to do so?

Additionally, we are in an awkward place where we have about four template dockstore.yml files of varying levels of complexity scattered across the docs. I was thinking of changing this template page I'm trying to link to into subpages, one for workflow .dockstore.yml, and one for service .dockstore.yml. It would not only give internal consistency but would also give a quick page for people to go to if they just need a .dockstore.yml template instead of all the info around it. What are your thoughts?

Reply:
Note that our live doc is on stable, not develop (just in case you're passing around https://docs.dockstore.org/en/develop links :) )

I think you want to use `:ref:` -- take a look at other examples.